Mesh_3/examples/Mesh_3/mesh_polyhedral_domain_with_features.cpp

```diff
@@ -32,8 +33,15 @@ using namespace CGAL::parameters;
 int main(int argc, char*argv[])
 {
   const char* fname = (argc>1)?argv[1]:"data/fandisk.off";
+  std::ifstream input(fname);
+  Polyhedron polyhedron;
+  input >> polyhedron;
+  if(input.bad()){
```

Comment:
It should be `input.fail()`. If the `failbit` is set, `fail()` returns `true`. If the `badbit` is set, `fail()` will also return `true`.